ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/modules/ToolboxModule.java

```diff
@@ -65,9 +63,7 @@ public abstract class ToolboxModule implements CloseableAndDisposable
    protected final String robotName;
    protected final FullHumanoidRobotModel fullRobotModel;
 
-   private final boolean manageROS2Node;
```

Comment:
This boolean seems unrelated to the ManagedROS2Node (despite its name). Should it really be removed?